std_msgs/Header header
float64 id
uint32 own_id
float64 x
float64 y
float64 z
float64 distance

